.github/workflows/workflows-issues-manager.yml

```diff
@@ -0,0 +1,85 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Workflows Issues Manager
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */12 * * *'
+  push:
+    branches: ['feat/issues-manager']
+
+jobs:
+  workflows_validation:
+    name: Workflows Issues Manager
+    permissions:
+      contents: write
+      pull-requests: write
+      checks: read
+      issues: read
+      statuses: read
+    # Don't run on forks
+#    if: github.repository == 'apache/beam'
```

Comment:
(a mark and remove the comment before merge)

Reply:
Noted 👍